lib/puma/request.rb

```diff
@@ -547,7 +547,7 @@ def str_headers(env, status, headers, res_body, io_buffer, force_keep_alive)
       colon = COLON
 
       resp_info = {}
-      resp_info[:no_body] = env[REQUEST_METHOD] == HEAD
+      resp_info[:no_body] = env[REQUEST_METHOD] == "HEAD"
```

Comment:
Probably still want to compare using a constant here?

Reply:
Sure, I'll bring it back. :wink: